arrows/vtk/applets/tests/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+include(kwiver-test-setup)
+
+add_test(NAME vtk:applets:estimate_depth
+  COMMAND kwiver estimate-depth
+            --config "${kwiver_test_data_directory}/estimate_depth_testing.conf"
+            --video-source "${kwiver_test_data_directory}/aphill_240_1fps_crf32.mp4"
+            --input-cameras-dir "${kwiver_test_data_directory}/krtd"
+            --input-landmarks-file "${kwiver_test_data_directory}/landmarks.ply")
+add_test(NAME vtk:applets:fuse_depth
```

Comment:
This test is failing on some CI machines that don't have CUDA because the default config uses CUDA.  We need to borrow some more code from TeleSculptor and update `fuse-depth` to work around this.  TeleSculptor has two different configs, [one for GPU](https://github.com/Kitware/TeleSculptor/blob/master/config/gui_integrate_depth_maps_gpu.conf) and [one for CPU](https://github.com/Kitware/TeleSculptor/blob/master/config/gui_integrate_depth_maps.conf).  They both include a [common set of parameters](https://github.com/Kitware/TeleSculptor/blob/master/config/integrate_depth_maps_common.conf) that works for both cases.  Then there is a [config option to use GPU](https://github.com/Kitware/TeleSculptor/blob/master/gui/tools/FuseDepthTool.cxx#L98-L102) that selects which config to load.  We can also [check if the cuda plugin](https://github.com/Kitware/TeleSculptor/blob/c7059fd18adad9246f21a8a3f75f835f857588cf/gui/MainWindow.cxx#L1731) is available and use that to set the default option to use GPU only if cuda is found.